docs/bpmn-support-roadmap.adoc

```diff
@@ -36,6 +36,7 @@ BPMN elements:
 ** [x] timer start event (0.1.3)
 ** [x] none end event (0.1.1)
 ** [x] terminate end event (0.1.2)
+** [x] none throw intermediate event (0.1.3)
```

Comment:
I didn't manage this one as we decided to moving to issue and milestone :smiley_cat: 
but as this is not in place, this is a good catch